src/build/sorter.ts

```diff
@@ -0,0 +1,41 @@
+import * as fs from "fs";
+import commentJson from "comment-json";
```

Comment:
We already use jsonc-parser, should be able to use here.

The important thing is that the comments should be there, while it's all being stripped out in this PR.

Reply:
I think we don't need to check in a full sorter, it should be enough to check whether it's sorted and throw error if not when linting.